src/main/java/com/patientpal/backend/post/controller/BoardController.java

```diff
@@ -0,0 +1,70 @@
+package com.patientpal.backend.post.controller;
+
+import com.patientpal.backend.member.domain.Member;
+import com.patientpal.backend.member.domain.Role;
+import com.patientpal.backend.member.service.MemberService;
+import com.patientpal.backend.post.domain.Post;
+import com.patientpal.backend.post.dto.*;
+import com.patientpal.backend.post.libs.RoleType;
+import com.patientpal.backend.post.service.PostService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'